lib/model/query/entities.js

```diff
@@ -415,13 +409,15 @@ const _processSubmissionEvent = (event, parentEvent) => async ({ Audits, Dataset
       throw Problem.user.entityActionNotPermitted({ action, permitted: permittedActions });
```

Comment:
This will probably need to change to prevent updates-applied-as-creates from happening on forms that dont allow creates. Double check when returning to force-processing work.